src/include/btmem.h

```diff
@@ -820,6 +820,17 @@ struct __wt_ref {
     WT_PAGE *volatile home;        /* Reference page */
     volatile uint32_t pindex_hint; /* Reference page index hint */
 
+    uint8_t __unused[2]; /* Padding */
```

Comment:
I suspect you moved this to the middle of the structure so that most accesses were somewhat aligned and don't need a memory copy. It's probably worth describing that in a comment, since it's not obvious why the padding should be in the middle of the structure.

Reply:
Two reasons: as you say, we wanted the 4 `uint8_t` fields in a row, after the `uint32_t pindex_hint` field for better alignment, and second, `unused` is specifically where it is so we can naturally grow the `WT_REF.flags` to 2 bytes, to add more bits if needed.

Pushed in 57e24884f.